### ISSUE TITLE
RuntimeWarning: Invalid value encountered in percentile   RuntimeWarning

### ISSUE THREAD

**Issue by piyushjhawar24** [pydata/pandas#14328]
#### A small, complete example of the issue
I am Using Pycharm
PyCharm Community Edition 2016.2.3
Build #PC-162.1967.10, built on September 7, 2016
JRE: 1.8.0_112-release-b343 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o

```python
arr = np.array([[1,2,np.nan],[np.nan,3,4]])
dframe1 = DataFrame(arr,index=['A','B'],columns=['One','Two','Three'])
print dframe1.describe()
```

When I run .describe function it throws me an exception

```
 .../anaconda/lib/python2.7/site-packages/numpy/lib/function_base.py:3834: RuntimeWarning: Invalid value encountered in percentile
  RuntimeWarning)
       One       Two  Three
count  1.0  2.000000    1.0
mean   1.0  2.500000    4.0
std    NaN  0.707107    NaN
min    1.0  2.000000    4.0
25%    NaN  2.250000    NaN
50%    NaN  2.500000    NaN
75%    NaN  2.750000    NaN
max    1.0  3.000000    4.0
```

#### Expected Output
```
       One       Two  Three
count  1.0  2.000000    1.0
mean   1.0  2.500000    4.0
std    NaN  0.707107    NaN
min    1.0  2.000000    4.0
25%    NaN  2.250000    NaN
50%    NaN  2.500000    NaN
75%    NaN  2.750000    NaN
max    1.0  3.000000    4.0
```

#### Output of ``pd.show_versions()``

<details>
# Paste the output here

</details>

**Comment by TomAugspurger** [comment]
What version of pandas are you using?

**Comment by jorisvandenbossche** [comment]
This was an issue in 0.18.0/0.18.1 (https://github.com/pydata/pandas/issues/13098), and is solved in master (and will be in the upcoming 0.19.0 (will be releases in one of the coming days)).
